gapis/api/gles/gles.api

```diff
@@ -119,11 +120,11 @@ class SupportedExtensions {
   bool GL_EXT_primitive_bounding_box                   = true
   bool GL_EXT_raster_multisample                       = true
   bool GL_EXT_robustness                               = true
-  bool GL_EXT_sRGB_write_control                       = true
```

Comment:
It does not matter now, just be aware that once we start using MEC, this class (and most state) will be append-only.

Reply:
Ack